feedback/cached.py

```diff
@@ -139,3 +148,78 @@ def get_obj(self, key, spec_getter, i18n_getter): # pylint: disable=unused-argum
 
 
 CachedForm = CachedForm(timeout=60*60)
+
+
+class MiscCache:
+    """Cache for storing miscellaneous content related to a course."""
+    def __init__(self, prefix=None, timeout=None) -> None:
+        self.prefix = prefix or self.__class__.__name__
+        self.timeout = timeout or 60 * 60
+
+    def get_suffix(self, *args) -> str:
+        return '-'.join(str(x) for x in args)
+
+    def get(self, key: str, course: Course):
+        full_key = '/'.join((self.prefix, self.get_suffix(key, course.id)))
+        return cache.get(full_key)
```

Comment:
Good point. I thought that since this return value type would be Any, it could be left out. But on further thought, as it is either None or a picklable object (as per [Django documentation](https://docs.djangoproject.com/en/5.0/topics/cache/#django.core.cache.cache.get)), the type could probably be defined as an [`object`](https://docs.djangoproject.com/en/5.0/topics/cache/#django.core.cache.cache.get)